### ISSUE TITLE
Some(1) must beNone fails to compile

### ISSUE THREAD

**Issue by GoogleCodeExporter** [etorreborre/specs#58]
```
===
Some(1) must beNone
===

fails to compile:

===
found   : org.specs.matcher.Matcher[Option[Nothing]]
required: org.specs.matcher.Matcher[Option[Int]]
===
```

Original issue reported on code.google.com by `stepan.k...@gmail.com` on 19 Feb 2009 at 7:51

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in 1.4.4-SNAPSHOT.

The workaround for this in 1.4.3 is to write: Some(1) must beNone[Int].
```

Original comment by `etorrebo...@gmail.com` on 19 Feb 2009 at 11:07
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 14 Feb 2010 at 2:43
- Changed state: **Verified**
